main.js

```diff
@@ -19,15 +19,19 @@ function Hoverable() {
 	var win = window;
```

Comment:
Now taht *is* an overoptimisation (I think I'm the guilty party originally). Will refactor it out